app/templates/signup.hbs

```diff
@@ -47,6 +47,7 @@
     {{/each}}
     {{#if this.isSubmitDisabled }}
       <button
+        data-test-id='signup-button-disabled'
```

Comment:
The state of the button changes, so we should not use that to identity the element here
```suggestion
        data-test-id='signup-submit-button'
```

Reply:
I see. This challenge that you faced comes from https://github.com/Real-Dev-Squad/website-my/pull/238#discussion_r930579530